### PR TITLE
Litmus Docs Fixes [Architecture Diagrams, Missing Operator and Runner Diagrams, and GitOps Clarification]

### LINKED ISSUE #147: Chaos Operator and Chaos Runner Diagrams not showing in v2.1 and v2.2 Docs
## BUG REPORT

**What Happened:** Chaos Operator and Chaos Runner Diagrams not showing in the `Architecture/Chaos Execution Plane` Docs for v2.1 and v2.2

**What You Expected to Happen:** Chaos Operator and Chaos Runner Diagrams should be appearing in the `Architecture/Chaos Execution Plane` Docs for v2.1 and v2.2, just as it is in master and v2.0

**How to Reproduce It (as minimally and precisely as possible):** Check `Architecture/Chaos Execution Plane` docs for v2.1 and v2.2

### LINKED ISSUE #148: Architecture Section Diagrams are Not Properly Visible in Dark Mode
## BUG REPORT

**What Happened:** Architecture Section Diagrams are Not Properly Visible in Dark Mode

**What you expected to happen**: Architecture section diagrams should have a white contrast to be properly visible in dark mode

**How to reproduce it (as minimally and precisely as possible)**: View the architecture section diagrams in dark mode.

### LINKED ISSUE #149: Clarity On the Behaviour of GitOps on Adding a new Workflow to the Repo
## FEATURE REQUEST

**What happened**: There is a Lack of Clarity On the Behaviour of GitOps on Adding a new Workflow to the Repo

**What you expected to happen**: The User Guide on GitOps should properly clarify the exact behaviour when a new workflow manifest is added to an existing repository with GitOps configured.

### PR DESCRIPTION
**Pull request by neelanjan00**
## What This PR Does
1. Adds background contrasts to the architecture docs diagrams to render them correctly in the dark mode.
2. Adds Chaos Operator and Chaos Runner diagrams to v2.1 and v2.2 docs
3. Adds clarification on the exact behavior when a workflow is pushed to a Git repository that has been configured for GitOps.

## Which Issues This PR Fixes
- Fixes #148 
- Fixes #147 
- Fixed #149 

**Checklist:**
-   [x] Fixes #<issue number>
-   [x] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/user-guides/gitops-configuration.md [hunk 1]
```diff
@@ -37,8 +35,9 @@ GitOps enables you to configure a single source of truth for your chaos workflow
 - Some metadata will be pushed to your repository, that is the projectID of your project.
 - Now whenever you schedule a workflow, it will automatically be pushed to your repository. And that repository will be the single source of truth.
 
-## Steps to configure Event-Triggered Chaos Injection
+**Note:** It is also possible to account for the workflows that are directly pushed to the git repository, after configuring GitOps. In this case, if the workflow is a single run workflow, then it starts executing as soon as it is pushed to the repository. Alternatively, if the workflow is a scheduled workflow, then it executes as per the defined schedule.
```
Comment (gdsoumya) [inline website/docs/user-guides/gitops-configuration.md]: Can we specify that "creating" a new workflow in git will execute the workflow whereas "updating" an existing workflow will not execute the workflow rather just update the workflow resource definition in the db and the cluster if applicable?

**Review by gdsoumya — APPROVED** [review]

**Comment by neelanjan00** [comment]
@gdsoumya can you PTAL? I have added another sentence to the note.

**Comment by neelanjan00** [comment]
@ksatchit can you PTAL and approve this PR?

**Comment by neelanjan00** [comment]
@ksatchit this PR is due for a long time now, can we merge this? I have updated it for v2.3 and v2.4.

**Review by ksatchit — APPROVED** [review]

File: website/versioned_docs/version-2.1.0/architecture/chaos-execution-plane.md [hunk 2]
```diff
@@ -33,10 +33,18 @@ Litmus Backend Execution Infrastructure components orchestrate the execution of
...
+</div>
+
 5. **ChaosResult CR:** Holds the results of a chaos experiment, such as ChaosEngine reference, Experiment State, Verdict of the experiment (on completion), salient application/result attributes. It also acts as a source for metrics collection for observability.
 
 6. **Chaos Runner:** Acts as a bridge between the Chaos Operator and Chaos Experiments. It is a lifecycle manager for the chaos experiments that creates Experiment Jobs for the execution of experiment business logic and monitors the experiment pods (jobs) until completion.
 
+<div style={{textAlign: 'center'}}>
+  <img src={require("../assets/chaos-execution-plane-chaos-runner.png").default} alt="Chaos Runner" />
+</div>
+
```
Comment (uditgaurav) [inline website/versioned_docs/version-2.1.0/architecture/chaos-execution-plane.md:47]: why the images are not visible here?

Comment (neelanjan00) [reply website/versioned_docs/version-2.1.0/architecture/chaos-execution-plane.md:47]: @uditgaurav They are visible in my local server:
Screenshot from 2022-01-14 20-15-14
Screenshot from 2022-01-14 20-15-21

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**